src/server.h

```diff
@@ -1422,6 +1422,8 @@ struct redisMemOverhead {
     float rss_extra;
     size_t rss_extra_bytes;
     size_t num_dbs;
+    size_t db_overhead_hashtable_lut;
+    unsigned long db_dict_rehashing_count;
     struct {
         size_t dbid;
```

Comment:
I think it seems that each db should also have its own information about `overhead_ht_lut`, `overhead_ht_rehashing`, and `dict_rehashing_count`. 

@oranagra WDYT?